climakitae/data/variable_descriptions.csv

```diff
@@ -34,8 +34,8 @@ wspd10max,maximum 10-meter wind speed,Dynamical,daily/monthly,m/s,Maximum wind s
 psfc,surface pressure,Dynamical,daily/monthly,hPa,Surface Pressure,Air pressure at Earth's surface.,ae_diverging,TRUE
 q2,2-meter specific humidity,Dynamical,daily/monthly,g/kg,Specific humidity at 2m,"The ratio of the mass of water vapor over the mass of total (moist) air, measured 2m above Earth's surface.",ae_blue,TRUE
 tskin,surface skin temperature ,Dynamical,daily/monthly,K,Surface skin temperature,Temperature of the Earth's surface (ground surface).,ae_orange,TRUE
-t2min,daily minimum 2-m temperature ,Dynamical,daily/monthly,K,Daily minimum air temperature at 2m,The minimum daily air temperature at 2m above the Earth's surface. ,ae_orange,TRUE
-t2max,daily maximum 2-m temperature ,Dynamical,daily/monthly,K,Daily maximum air temperature at 2m,The maximum daily air temperature at 2m above the Earth's surface. ,ae_orange,TRUE
+t2min,daily minimum 2-m temperature ,Dynamical,daily/monthly,K,Minimum Air Temperature at 2m,The minimum air temperature at 2m above the Earth's surface. ,ae_orange,TRUE

```

Comment:
Should the long_name also be modified here as well? "Daily" is still included for both min/max

Reply:
Nah the long name is what is set by whoever generated the data-- can't be modified here and the user won't see it anyways.